ruby/ql/lib/codeql/ruby/typetracking/TypeTrackerSpecific.qll

```diff
@@ -11,8 +11,23 @@ class Node = DataFlowPublic::Node;
 
 class TypeTrackingNode = DataFlowPublic::LocalSourceNode;
 
-predicate simpleLocalFlowStep = DataFlowPrivate::localFlowStepTypeTracker/2;
+/** Holds if there is a simple local flow step from `nodeFrom` to `nodeTo` */
+predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
+  DataFlowPrivate::localFlowStepTypeTracker(nodeFrom, nodeTo)
+  or
+  // Summarize calls with flow directly from a parameter to a return.
+  exists(DataFlowPublic::ParameterNode param, DataFlowPrivate::ReturningNode returnNode |
+    DataFlowPrivate::LocalFlow::getParameterDefNode(param.getParameter())
+        .(TypeTrackingNode)
+        .flowsTo(returnNode) and
+    // Step from captured parameter to local call sites
```

Comment:
I don't understand this comment; it has nothing to do with variable capturing, right?

Reply:
That's just a left-over originating from https://github.com/github/codeql/blob/main/javascript/ql/lib/semmle/javascript/dataflow/internal/StepSummary.qll#L154